spynnaker/pyNN/models/common/neuron_recorder.py

```diff
@@ -239,13 +375,14 @@ def get_spikes(
                 bits = numpy.fliplr(numpy.unpackbits(spikes).reshape(
                     (-1, 32))).reshape((-1, n_bytes * 8))
                 time_indices, local_indices = numpy.where(bits == 1)
-                if self.__indexes[SPIKES] is None:
+                if self.__indexes[self.SPIKES] is None:
```

Comment:
this opens the whole issue of what we do with mapping between named variables and types. I've stored it as self.SPIKES because previously it was just the text "spikes" which was shitter.  We need to think how we should seperate a region name and its type. but thats a later discussion than this work